jenkins/ci.suse.de/pipelines/openstack-crowbar.Jenkinsfile

```diff
@@ -217,6 +217,17 @@ pipeline {
       }
     }
 
+    stage('Update cloud') {
+      when {
+        expression { deploy_cloud == 'true' && update_after_deploy == 'true' }
```

Comment:
You also need a `update_after_deploy` parameter to be added to the `openstack-crowbar` JJB template.

Reply:
done